src/routes/+layout.style.scss

```
@@ -190,11 +190,11 @@ main {
   position: fixed;
   top: 0;
   left: 0;
-  z-index: 10;
+  z-index: 100;
   padding: 0.75rem 1rem;
   background: var(--color-background);
   color: var(--color-foreground);
-  outline: 2px solid currentColor;
+  border: 2px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)